prow/config/config_test.go

```diff
@@ -146,6 +146,28 @@ deck:
 			expectedSizeLimit: 500e6,
 			expectError:       false,
 		},
+		{
+			name: "Backwards compatibility",
+			spyglassConfig: `
+deck:
+  spyglass:
+    size_limit: 500e6
```

Comment:
This is wrong and should be `500e+6`. (https://github.com/kubernetes/test-infra/pull/10266/commits/44b0296f8b6ca85790270bd3d1f825cb919a3f8f)

Reply:
I hate yaml so much.

(shout out to the yaml spec for making the same mistake!)

Reply:
(it was incorrect behaviour before)

Reply:
I agree, it's awful. :(